index.html

```diff
@@ -1,11 +1,48 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Product Landing Page</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Landing Page</title>
+    <link rel="stylesheet" href="landingPage.css">
 </head>
 <body>
-  
+  <!-- Navigation bar -->
+    <header>
+        <nav>
+            <a href="#Home"><img src="logo.png" alt="Home"></a>
+            <ul>
+                <li><a href="#Home">Home</a></li>
+                <li><a href="#Shop">Shop</a></li>
```

Comment:
**[Suggestion]**
Since you have `#faq`, `#contact` at the bottom perhaps we should make these links consistent too, e.g. `#home`.

Reply:
fixed